stretch_install_user.sh

```diff
@@ -1,15 +1,27 @@
 #!/bin/bash
 
-#Current issues
-# python -m pip install --user numba
-# catkin  - csm
-
-
-echo "Setting up Stretch new user account"
 
+if [ "$(ls -A ~/stretch_user)" ]; then
+    echo "###########################################"
+    echo "UPDATING USER SOFTWARE"
+    UPDATING=true
+else
+    UPDATING=false
+    echo "###########################################"
+    echo "NEW INSTALLATION OF USER SOFTWARE"
+    echo "UPDATE .bashrc FOR STRETCH_BODY"
+    echo "export EDITOR='emacs -nw'" >> ~/.bashrc
```

Comment:
Users might not want Emacs. I switched to vim.

Reply:
This was carried forward from CK's script long ago. Do we even need it?

Reply:
No, we can get rid of it. Users can choose their preferred terminal editor.

Reply:
Removed it.